bittide-shake/data/tcl/HardwareTest.tcl

```diff
@@ -220,9 +220,10 @@ proc get_ila_dicts {} {
         set ila_dict {}
 
         set cell_name [get_property CELL_NAME $hw_ila]
-        set idx_start [expr {[string first _ $cell_name] + 1}]
-        set idx_end [expr {[string first / $cell_name] - 1}]
-        set short_name [string range $cell_name $idx_start $idx_end]
+        set after_last [expr [string last "/" $cell_name] + 1]
+        set ila_name_suffix [string range $cell_name $after_last end]
+        set under_idx [expr [string first _ $ila_name_suffix] + 1]
+        set short_name [string range $ila_name_suffix $under_idx end]
```

Comment:
I don't get how this matches with the examples of the PR description. As far as I understand these lines of code, they first drop all the prefix up till the last `/` and then the prefix up till the first `_`. But then
```
Bittide_Instances_Hitl_FullMeshSwCc_fullMeshSwCcTest_ilaWb58_result_22/ila_inst
```
would result in `inst.csv` and not `Instances_Hitl_FullMeshSwCc_fullMeshSwCcTest_ilaWb58_result_22.csv`, right? 

I guess that I am just missing a bit here. Can you help me out?

Reply:
You are completely right. This used to be present because of the example I gave [here](https://github.com/bittide/bittide-hardware/pull/456#issuecomment-1910118282), but I don't see any ILA cell names with the structure `{module-name}_{setName}/{module-name}_{setName}` which I explained there. I don't really understand why those aren't there anymore. But I will remove the commit where I change the Tcl, and only keep your initial commit.

Reply:
Neither of our solutions work, specifically because of the 2 ILAs in `processingElement`. Because they both get called the same name one will overwrite the data dump of the other...